src/client/features/view/cy/events/hover.js

```diff
@@ -94,9 +94,15 @@ const baseEdgeHoverStyle = {
 };
 
 const bindHover = (cy) => {
-  cy.on('mouseover', 'node[class!="compartment"]', function (evt) {
+  cy.on('mouseover', 'node[class !="compartment"]', function (evt) {
     const node = evt.target;
     const currZoom = cy.zoom();
+    const isCollapsed = node.data('compoundCollapse.collapsed');
```

Comment:
This can be simplified to:
```js
    if (node.isParent() && node.isExpanded()) { return; }
```

http://js.cytoscape.org/#node.isParent
https://github.com/d2fong/cytoscape.js-compound-collapse#expand

Reply:
Deleted isCollapsed and used isExpanded()